### PR TITLE
ContraNovo test fix

### PR DESCRIPTION
**Pull request by aukeheerdink**

### CODE REVIEW

File: rustyms/src/identification/identified_peptide.rs [hunk 1]
```diff
@@ -385,9 +385,10 @@ impl IdentifiedPeptide {
             }
 
             MetaData::PowerNovo(PowerNovoData { raw_file, scan, .. }) => {
-                scan.clone().map_or(SpectrumIds::None, |scan| {
-                    SpectrumIds::FileKnown(vec![(raw_file.clone(), vec![scan])])
-                })
+                let raw_file = raw_file.as_ref().unwrap();
```
Comment (douweschulte) [inline rustyms/src/identification/identified_peptide.rs]: .unwrap() means a possibility for a crash. Please use the appropriate `SpectrumIds::None/FileUnknown` to return the absence of those parts of information.

File: rustyms/src/identification/instanovo.rs [hunk 2]
```diff
@@ -52,7 +52,13 @@ impl From<InstaNovoData> for IdentifiedPeptide {
     fn from(value: InstaNovoData) -> Self {
         Self {
             score: Some(2.0 / (1.0 + 1.01_f64.powf(-value.score))),
-            local_confidence: Some(value.local_confidence.iter().map(|v| *v / 100.0).collect()),
+            local_confidence: Some(
+                value
+                    .local_confidence
+                    .iter()
+                    .map(|v| 2.0 / (1.0 + (-v).exp()))
```
Comment (douweschulte) [inline rustyms/src/identification/instanovo.rs]: Is it intended for this calculation to differ from the one above (line 54 in my view) where the full score is normalised?

**Review by douweschulte — COMMENTED** [review]
Nice work. But I have two small questions remaining.

File: rustyms/src/identification/instanovo.rs (hunk 2 above)
Comment (aukeheerdink) [reply rustyms/src/identification/instanovo.rs]: Yes this is intentional, when I use the other calculation you get very misrepresented values. The current implementation represents the full range better.

File: rustyms/src/identification/identified_peptide.rs (hunk 1 above)
Comment (aukeheerdink) [reply rustyms/src/identification/identified_peptide.rs]: Ah yes apologies, was still testing some bits, I'll change that.

**Review by douweschulte — APPROVED** [review]

**Pull request merged**